### ISSUE TITLE
keyboard only

### ISSUE THREAD

**Issue by romevp** [ti-yu/sopra-fs26-group-19-server#16]
Story: As a user who is physically unable to use a mouse, I want the website to be navigable by a keyboard only, so I have equal access to the service the site provides 

Acceptance Criteria: 

- [ ] The entire site is navigable by keyboard 
- [ ] The navigation is intuitive and can be done without physical strain 
- [ ] How to navigate is clearly communicated to the user 
- [ ] Links and other elements within text blocks can be accessed via keyboard 
- [ ] The website adheres to WCAG2.2 Guideline 2.1 

Priority: low 

Author: Timur 

Estimate: 5h
